make/pre-commit.mk

```diff
@@ -11,4 +11,5 @@ ifeq ($(wildcard $(PRE_COMMIT_CONFIG_FILE)),)
 	$(error Cannot find pre-commit config file $(PRE_COMMIT_CONFIG_FILE). Specify the config file via PRE_COMMIT_CONFIG_FILE variable)
 endif
 	env SKIP=$(SKIP) pre-commit run -a --show-diff-on-failure --config $(PRE_COMMIT_CONFIG_FILE)
-	git fetch origin main && gitlint --ignore-stdin --commits origin/main..HEAD
+	git fetch origin release-2.4 && gitlint --ignore-stdin --commits origin/release-2.4..HEAD
+
```

Comment:
pre-commit is complaining about this extra line

Reply:
aye - fix incoming